sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/ExternalTypescriptSensor.java

```diff
@@ -281,6 +284,17 @@ private void saveHighlights(SensorContext sensorContext, Highlight[] highlights,
     highlighting.save();
   }
 
+  private void saveUsages(SensorContext sensorContext, Symbol[] symbols, InputFile inputFile) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make "saveUsages" a "static" method. ([squid:S2325](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2325))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=sonarts&branch=stas%2Fsymbol-highlighting&issues=AWCdvfnyqDrCo5ysf9-v&open=AWCdvfnyqDrCo5ysf9-v)

[ISSUE_KEY]: AWCdvfnyqDrCo5ysf9-v